### PR TITLE
profile picture update

### PR DESCRIPTION
**Pull request by erdoganb**
profile picture update
add picture
resize picture
create a random name for uploaded picture
create path 
assign to current_user in database

updated account page for account page
added AccountUpdateForm to forms.py

### CODE REVIEW

File: flaskblog/forms.py [hunk 1]
```diff
@@ -29,4 +30,23 @@ class LoginForm(FlaskForm):
...
+            user = User.query.filter_by(username=username.data).first()
+            if user:
+                raise ValidationError('That username already taken. Try different one.')
+
+    
+    def validate_email(self, email):
+        if email.data != current_user.email:
+            user = User.query.filter_by(email=email.data).first()
+            if user:
+                raise ValidationError('That email already taken. Try Differen one.')
```
Comment (OguzOzkeroglu) [inline flaskblog/forms.py]: typo: Differen -> different

File: run.py [hunk 2]
```diff
@@ -3,4 +3,4 @@
 
 
 if __name__ == '__main__':
-    app.run(debug=False)
+    app.run(debug=True)
```
Comment (OguzOzkeroglu) [inline run.py]: Should be False

**Review by OguzOzkeroglu — CHANGES_REQUESTED** [review]

File: flaskblog/forms.py (hunk 1 above)
Comment (erdoganb) [reply flaskblog/forms.py]: bunu nası gördün amk ya

**Review by OguzOzkeroglu — APPROVED** [review]
good job

**Pull request merged**